ghclient/core.go

```diff
@@ -0,0 +1,147 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package ghclient provides a github client that wraps go-github with retry logic, rate limiting,
```

Comment:
Is this client supposed to be used by everything now? Are we going to collapse with prow eventually?

Reply:
Its a common client that can be used for most all of our purposes. @spxtr wants Prow to have its own github client.

Reply:
ack